MainDemo.Wpf/Chips.xaml.cs

```diff
@@ -23,16 +25,17 @@ public partial class Chips : UserControl
         public Chips()
         {
             InitializeComponent();
+			
         }
 
         private void ButtonsDemoChip_OnClick(object sender, RoutedEventArgs e)
         {
-            Console.WriteLine("Chip clicked.");
+            MessageBox.Show("Chip clicked.");
         }
 
         private void ButtonsDemoChip_OnDeleteClick(object sender, RoutedEventArgs e)
         {
-            Console.WriteLine("Chip delete clicked.");
+            MessageBox.Show("Chip delete clicked.");
```

Comment:
If we are going to amend this I'd prefer we use the top level SnackBar.

Reply:
Sure

Reply:
Done. Refer c2b6dd3110a4f8bf2404d3a07d8245830a5dfee5